addons/binding/org.openhab.binding.harmonyhub/src/main/java/org/openhab/binding/harmonyhub/handler/HarmonyHubHandler.java

```diff
@@ -209,7 +216,26 @@ private synchronized void connect() {
         discoTime = config.discoveryTimeout > 0 ? config.discoveryTimeout : DISCO_TIME;
         heartBeatInterval = config.heartBeatInterval > 0 ? config.heartBeatInterval : HEARTBEAT_INTERVAL;
 
-        discovery = new HarmonyHubDiscovery(discoTime);
+        String host = config.host;
+
+        // earlier versions required a name and used network discovery to find the hub and retrieve the host,
```

Comment:
> I see the "method":"pair" in the auth message, where did you find that?

The harmony-java-client command line interface previously used a REST API to obtain the session ID. For some reason this REST API no longer works. I found this workaround being used in:

* [Domoticz](https://github.com/domoticz/domoticz/blob/master/hardware/HarmonyHub.cpp#L480)
* another [fork](https://github.com/tuantran/harmony-java-client/blob/master/src/main/java/net/whistlingfish/harmony/protocol/MessageAuth.java#L31) of the harmony-java-client

It simplifies authentication and also allows for removing the dependency on RESTY which was previously used to consume the REST API.